tests/model/test_labels.py

```
@@ -1,6 +1,6 @@
 """Test methods and functions in the sleap_io.model.labels file."""
 
-from numpy.testing import assert_equal
+from numpy.testing import assert_equal, assert_allclose
```

Ruff

`numpy.testing.assert_equal` imported but unused

Remove unused import: `numpy.testing.assert_equal`

(F401)